import_manager.go

```diff
@@ -90,10 +90,9 @@ sendRecords:
 	return importErr
 }
 
-func recordImportWorker(id int, client *Client, field *Field, chans importWorkerChannels, options ImportOptions) {
+func recordImportWorker(id int, client *Client, field *Field, chans importWorkerChannels, options *ImportOptions) {
```

Comment:
Not that I have an objection, but why did ImportOptions become a pointer here? Is this just because in general convention is to use pointers for structs that aren't specifically known to be very small?

Reply:
Yeah, I thought not-copying `ImportOptions` would be fine there since it is not mutated anywhere. I had to move `importRecords` to its global function, since goreportcard was complaining that `recordImportWorker` became too complex.